step5.html

```diff
@@ -30,34 +30,14 @@
 rake -T
 {% endhighlight %}
 <p>
-It failed!&hellip; and with a rather cryptic error message. But don't worry, it's
-easy to fix.
-</p>
-<p>
-Many of the tasks provided by yast-rake are related to packaging (to be precise,
-those packaging tasks are actually provided by the
-<a href="https://rubygems.org/gems/packaging_rake_tasks">packaging_rake_tasks</a>
-gem, required by yast-rake). For that reason you need a package specification to
-get the YaST rake tasks running. In your case, that is as easy as executing the
-following command.
+Now, let's try some of the useful available tasks. But first we need to install
+"rubygem-gettex".
 </p>
 {% highlight bash %}
-git checkout package_spec
+zypper install ruby2.1-rubygem-gettext
 {% endhighlight %}
 <p>
-Now, &quot;package/yast2-journalctl.spec&quot; contains a basic package
-specification to be processed by the
-<a href="http://openbuildservice.org/">Open Build Service</a> (mainly a 
-<a href="http://www.rpm.org/">RPM</a> spec file). We will go back to packaging
-in the last step of the tutorial, but if you want to learn more about the Open
-Build Service and openSUSE packages creation, check the
-<a href="https://en.opensuse.org/Portal:Packaging">corresponding portal</a>
-in the openSUSE wiki.
-</p>
```

Comment:
Maybe the content of this paragraph should be moved to the corresponding section about packaging. I'll check if the information is there already.